### DISCUSSION TITLE
what's the equivalent usage with the { version: 3, proposals: true }?

### DISCUSSION THREAD

**Discussion by mrdulin** [zloirock/core-js#1349] [Q&A] [answered]
```json
["@babel/preset-env", { "useBuiltIns": "usage", "corejs": { "version": "3.8", "proposals": true } }]
```
what's the equivalent usage with above? I want to add the polyfill by myself.

```js
import "core-js";
// or
import "core-js/full";
```

**Comment by zloirock** [comment, marked as answer]
`proposals` option makes no sense for `useBuiltIns: entry` since that replaces the import of any `core-js` entry to the import only required modules for your target environments.

In `core-js@3`, `import "core-js"` is a full equal of `import "core-js/full"`. In `core-js@4` it will be changed and `import "core-js"` will be equal of `import "core-js/actual"` ("stable" + stage 3 proposals).

It seems your question is related to `useBuiltIns: usage` and `core-js@3`. In this case, with `proposals: true` used `core-js` / `core-js/full` subset of features, with `proposals: false` (default) - `core-js/stable` subset.

**Reply by mrdulin** [reply]
Yes. I updated the question, it's `useBuiltIns: usage`
